FluidNC/src/Machine/MachineConfig.cpp

```diff
@@ -94,36 +94,36 @@ namespace Machine {
     }
 
     void MachineConfig::groupM4Items(Configuration::HandlerBase& handler) {
-        handler.item("Maslow_vertical", Maslow.orientation);
-        handler.item("maslow_calibration_grid_width_mm_X", Maslow.calibration_grid_width_mm_X, 100, 3000);
-        handler.item("maslow_calibration_grid_height_mm_Y", Maslow.calibration_grid_height_mm_Y, 100, 3000);
-        handler.item("maslow_calibration_grid_size", Maslow.calibrationGridSize, 3, 9);
+        handler.item(M+"_vertical", Maslow.orientation);
+        handler.item(M+"_calibration_grid_width_mm_X", Maslow.calibration_grid_width_mm_X, 100, 3000);
+        handler.item(M+"_calibration_grid_height_mm_Y", Maslow.calibration_grid_height_mm_Y, 100, 3000);
+        handler.item(M+"_calibration_grid_size", Maslow.calibrationGridSize, 3, 9);
 
-        handler.item("Maslow_tlX", Maslow.tlX);
-        handler.item("Maslow_tlY", Maslow.tlY);
+        handler.item(M+"_tlX", Maslow.tlX);
+        handler.item(M+"_tlY", Maslow.tlY);
 
-        handler.item("Maslow_trX", Maslow.trX);
-        handler.item("Maslow_trY", Maslow.trY);
+        handler.item(M+"_trX", Maslow.trX);
+        handler.item(M+"_trY", Maslow.trY);
 
-        handler.item("Maslow_blX", Maslow.blX);
-        handler.item("Maslow_blY", Maslow.blY);
+        handler.item(M+"_blX", Maslow.blX);
+        handler.item(M+"_blY", Maslow.blY);
         
-        handler.item("Maslow_brX", Maslow.brX);
-        handler.item("Maslow_brY", Maslow.brY);
+        handler.item(M+"_brX", Maslow.brX);
+        handler.item(M+"_brY", Maslow.brY);
 
-        handler.item("Maslow_tlZ", Maslow.tlZ);
-        handler.item("Maslow_trZ", Maslow.trZ);
-        handler.item("Maslow_blZ", Maslow.blZ);
-        handler.item("Maslow_brZ", Maslow.brZ);
+        handler.item(M+"_tlZ", Maslow.tlZ);
```

Comment:
I kind of imagined that the compiler would optimize out these strings, is that not the case?